sygnal/gcmpushkin.py

```diff
@@ -325,7 +349,7 @@ async def _dispatch_notification_unlimited(self, n, device, context):
             }
 
             # TODO: Implement collapse_key to queue only one message per room.
-            failed = []
+            failed: list = []
```

Comment:
```suggestion
            failed: List[str] = []
```
iirc this is a list of pushkeys which are strings :)